src/Database/Type/DateTimeType.php

```diff
@@ -257,7 +245,8 @@ public function marshal($value): ?DateTimeInterface
             } elseif ($isString && $this->_useLocaleMarshal) {
                 return $this->_parseLocaleValue($value);
             } elseif ($isString) {
-                return $this->_parseValue($value);
+                /** @var \DateTimeInterface */
+                return new $class($value);
```

Comment:
Won't this also enable users to send things like `+2 minutes -1 days` in API requests? Previously that would fail marshalling but would now pass.

Reply:
Yes.  It would pass at this level.

Reply:
It would, I am unsure if allowing relative time strings when marshalling is a good idea.

Reply:
Does it make a difference?

Reply:
I am unsure :)   But not having to maintain a long list of valid formats is certainly nice.

Reply:
I think it isn't desirable as it changes the behavior of application API endpoints to accept data that is not expected. Is the big list of formats hard to maintain?

Reply:
If the big list is hard to maintain could we do a quick pattern check on the input data to ensure it quacks like a datetime string?

Reply:
That is fine.  I just put this up to follow up on the idea.

Looks like I can use a single time zone format to catch: +0200, +02:00 and Europe/Paris.

That'll make the longest marshal list 8 formats (see below).  Shouldn't be too bad for now since it's just processing 1-3 inputs at most likely per request.